lib/features/manage_account/presentation/extensions/tmail_forward_extension.dart

```diff
@@ -1,12 +1,13 @@
 
 import 'package:forward/forward/tmail_forward.dart';
+import 'package:jmap_dart_client/jmap/mail/email/email_address.dart';
 import 'package:tmail_ui_user/features/manage_account/presentation/model/recipient_forward.dart';
 
 extension TMailForwardExtension on TMailForward {
 
   List<RecipientForward> get listRecipientForward {
     return forwards
-        .map((emailAddress) => RecipientForward(emailAddress))
+        .map((emailAddress) => RecipientForward(EmailAddress(emailAddress, emailAddress)))
```

Comment:
Why do you have to pass the same email to EmailAddress instead of 1 ?

Reply:
`emailAddress` is a string, here we need to reduce it to the `EmailAddress` object for synchronization.